### PR TITLE
Math png to text

### PR DESCRIPTION
**Pull request by kblaszczak-intel**
### Changes
mathematical equations in "compression_algorithms" have been changed from png to text, using LaTex formatting, which is rendered by GitHub natively and can be processed by Sphinx using Mathjax.

### Reason for changes
images are not visible in darkmode and are not searchable

### Related tickets
81822

### CODE REVIEW

File: docs/compression_algorithms/Pruning.md [hunk 1]
```diff
@@ -167,14 +167,14 @@ The columns `Full` and `Current` represent the values of the corresponding stati
 
 The `Pruning level` column indicates the ratio between the values of the full and current statistics in the corresponding rows, defined by the formula:
 
-![Statistic pruning level](https://latex.codecogs.com/png.image?\dpi{110}\textit{statistic&space;pruning&space;level}&space;=&space;1&space;-&space;\textit{statistic&space;current}&space;/&space;\textit{statistic&space;full})
+$Statistic pruning level = 1 - statistic current / statistic full$
```
Comment (vshampor) [inline docs/compression_algorithms/Pruning.md:170]: Please fix the spaces.

**Review by vshampor — CHANGES_REQUESTED** [review]
Great job, this helps a lot! There are minor issues to be corrected, though.

File: docs/compression_algorithms/Pruning.md [hunk 2]
```diff
@@ -167,14 +167,14 @@ The columns `Full` and `Current` represent the values of the corresponding stati
...
   
 `Filter pruning level` - percentage of filters removed from the model.  
 
 `GFLOPs pruning level` - an estimated reduction in the number of floating point operations of the model.   
 The number of FLOPs for a single convolutional layer can be calculated as:
 
-![FLOPs](https://latex.codecogs.com/png.image?%5Cdpi%7B110%7D%20FLOPs&space;=&space;2&space;%5Ctimes&space;input%5C;channels&space;%5Ctimes&space;kernel%5C;size%5E%7B2%7D%5Ctimes&space;W%5Ctimes&space;H%5Ctimes&space;filters)
+$FLOPs = 2x input channels x kernel size ^2 x W x H x filters$
```
Comment (kblaszczak-intel) [inline docs/compression_algorithms/Pruning.md:177]:
```suggestion
$FLOPs = 2 * input channels * kernel size ^2 * W * H * filters$
```
I hope this is better :)

File: docs/compression_algorithms/Quantization.md [hunk 3]
```diff
@@ -178,22 +191,22 @@ trace value are quantized to lower bitwidth and vice versa.
...
 Given Rademacher distributed random vector v, the trace of symmetric matrix H is equal to the estimation of a quadratic form:
 
-![Tr(H) = \mathbb{E}[v^T H v]](https://latex.codecogs.com/png.latex?Tr%28H%29%20%3D%20%5Cmathbb%7BE%7D%5Bv%5ET%20H%20v%5D)
+$Tr(H) = \mathbb{E}[v^T H v]$
 
 The randomized algorithm solves the expectation by Monte Carlo using sampling of v from its distribution, evaluating
 the quadratic term, and averaging:
 
-![Tr(H) \approx \frac{1}{m}\sum_{i=1}^{m}[v_i^T H v_i]](https://latex.codecogs.com/png.latex?Tr%28H%29%20%5Capprox%20%5Cfrac%7B1%7D%7Bm%7D%5Csum_%7Bi%3D1%7D%5E%7Bm%7D%5Bv_i%5ET%20H%20v_i%5D)
+$Tr(H) \approx \frac{1}{m}\sum_{i=1}^{m}[v_i^T H v_i]$
```
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/compression_algorithms/Quantization.md:199]:
```suggestion
$Tr(H) \approx \frac{1}{m}\sum\limits_{i=1}^{m}[v_i^T H v_i]$
```

File: docs/compression_algorithms/Pruning.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/compression_algorithms/Pruning.md]: "x" changed to "*"

File: docs/compression_algorithms/Pruning.md (hunk 1 above)
Comment (kblaszczak-intel) [reply docs/compression_algorithms/Pruning.md:170]:
```suggestion
$Statistic\\:pruning\\:level = 1 - statistic\\:current / statistic\\:full$
```

File: docs/compression_algorithms/Pruning.md [hunk 4]
```diff
@@ -167,14 +167,14 @@ The columns `Full` and `Current` represent the values of the corresponding stati
...
-![Statistic pruning level](https://latex.codecogs.com/png.image?\dpi{110}\textit{statistic&space;pruning&space;level}&space;=&space;1&space;-&space;\textit{statistic&space;current}&space;/&space;\textit{statistic&space;full})
+$Statistic\\:pruning\\:level = 1 - statistic\\:current / statistic\\:full$
   
 `Filter pruning level` - percentage of filters removed from the model.  
 
 `GFLOPs pruning level` - an estimated reduction in the number of floating point operations of the model.   
 The number of FLOPs for a single convolutional layer can be calculated as:
 
-![FLOPs](https://latex.codecogs.com/png.image?%5Cdpi%7B110%7D%20FLOPs&space;=&space;2&space;%5Ctimes&space;input%5C;channels&space;%5Ctimes&space;kernel%5C;size%5E%7B2%7D%5Ctimes&space;W%5Ctimes&space;H%5Ctimes&space;filters)
+$FLOPs = 2 * input channels * kernel size ^2 * W * H * filters$
```
Comment (kblaszczak-intel) [inline docs/compression_algorithms/Pruning.md:177]:
```suggestion
$FLOPs = 2 * input\\:channels * kernel\\:size ^2 * W * H * filters$
```

File: docs/compression_algorithms/Pruning.md [hunk 5]
```diff
@@ -185,8 +185,8 @@ significantly from the filter pruning level.
 In addition, the decrease in GFLOPs is estimated by calculating the number of FLOPs of convolutional and fully connected layers. 
 As a result, these estimates may differ slightly from the actual number of FLOPs in the compressed model.
 
-`MParams  pruning level` - calculated reduction in the number of parameters in the model in millions. Typically convolutional layer weights have shape 
-![shape](https://latex.codecogs.com/png.image?%5Cdpi%7B110%7D%20%5Cinline%20(kernel%5C;size,&space;kernel%5C;size,&space;input%5C;channels,&space;filters%5C;&space;num)).
+`MParams  pruning level` - calculated reduction in the number of parameters in the model in millions. Typically convolutional layer weights have the shape of $(kernel size, kernel size, input channels, filter num)$.
```
Comment (kblaszczak-intel) [inline docs/compression_algorithms/Pruning.md:188]:
```suggestion
`MParams  pruning level` - calculated reduction in the number of parameters in the model in millions. Typically convolutional layer weights have the shape of $(kernel\\:size,\\:kernel\\:size,\\:input\\:channels,\\:filter\\:num)$.
```

File: docs/compression_algorithms/Quantization.md [hunk 6]
```diff
@@ -155,7 +168,7 @@ configuration by taking into account the sensitivity of each layer, i.e. how muc
 decreases the accuracy of model. The most sensitive layers are kept at higher precision. The sensitivity of the i-th layer is
 calculated by multiplying the average Hessian trace with the L2 norm of quantization perturbation:
 
-![\overline{Tr}(H_{i}) * \left \| Q(W_{i}) - W_{i} \right \|^2_2](https://latex.codecogs.com/png.latex?%5Coverline%7BTr%7D%28H_%7Bi%7D%29%20*%20%5Cleft%20%5C%7C%20Q%28W_%7Bi%7D%29%20-%20W_%7Bi%7D%20%5Cright%20%5C%7C%5E2_2)
+$\overline{Tr}(H_{i}) * \left \| Q(W_{i}) - W_{i} \right \|^2_2$
```
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/compression_algorithms/Quantization.md:171]:
```suggestion
$\overline{Tr}(H_{i}) * \left \|\| Q(W_{i}) - W_{i} \right \|\|^2_2$
```

File: docs/compression_algorithms/Pruning.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/compression_algorithms/Pruning.md]: spaces added... Apparently this is required for a space between words "\\:"

**Comment by kblaszczak-intel** [comment]
Thanks for noticing the bugs. I hope now it is better.

**Pull request merged**